### PR TITLE
connected components in an undirected graph

### PR DESCRIPTION
**Pull request by veenit1802**
I have completed the code for the connected components in an undirected graph. Kindly, please check if there is any problems.
@tarun26091999 for issue-#531

### CODE REVIEW

File: Code/C++/connected_component_in_undirected_graph.cpp [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    visited[node]=true;
+    components[node]=cnt;
+    for(int curnode:graph_list[node])
+    {
+        if(!visited[curnode])
+        {
+            dfs(graph_list,visited,cnt,curnode,components);
+        }
+    }
+    return ;
```
Comment (tarun26091999) [inline Code/C++/connected_component_in_undirected_graph.cpp:38]: remove this line.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link it to the appropriate folder.

**Review by veenit1802 — COMMENTED** [review]
@tarun26091999 sir changes are made kindly check it

**Comment by veenit1802** [comment]
Sir, the conflict in the file is resolved, kindly check it. @tarun26091999

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
remove -> sign after the last vertex of the component.

**Pull request closed without merging**